jck/runtime.vm/playlist.xml

```diff
@@ -104,6 +98,12 @@
 				<impl>ibm</impl>
 				<version>11</version>
 			</disable>
+			<disable>
+				<comment>Temporarily disabled on z/OS for backlog/issues/670</comment>
+				<platform>.*zos.*</platform>
+				<impl>ibm</impl>
+				<version>8</version>
+			</disable>
```

Comment:
Instead of adding a new `<disable>`, please update line 99 to `<version>8+</version>`